tests/text/test_ter.py

```diff
@@ -0,0 +1,187 @@
+from functools import partial
+from typing import Sequence
+
+import pytest
+from torch import Tensor, tensor
+
+from tests.text.helpers import INPUT_ORDER, TextTester
+from torchmetrics.functional.text.ter import ter
+from torchmetrics.text.ter import TER
+from torchmetrics.utilities.imports import _SACREBLEU_AVAILABLE
+
+if _SACREBLEU_AVAILABLE:
+    from sacrebleu.metrics import TER as SacreTER
+
+# example taken from
+# https://www.nltk.org/api/nltk.translate.html?highlight=bleu%20score#nltk.translate.bleu_score.corpus_bleu and adjusted
+# EXAMPLE 1
+HYPOTHESIS_A = "It is a guide to action which ensures that the military always obeys the commands of the party"
+REFERENCE_1A = "It is a guide to action that ensures that the military will forever heed Party commands"
+REFERENCE_2A = "It is a guiding principle which makes the military forces always being under the command of the Party"
+
+# EXAMPLE 2
+HYPOTHESIS_B = "he read The Book because he was interested in World history"
+REFERENCE_1B = "he was interested in world history because he read the book"
+REFERENCE_2B = "It is the practical guide for the army always to heed the directions of the party"
+
+# EXAMPLE 3 (add intentionally whitespaces)
+HYPOTHESIS_C = "the cat the   cat on the mat "
+REFERENCE_1C = "the  cat is     on the mat "
+REFERENCE_2C = "there is a   cat on the mat"
```

Comment:
just thinking, shall we move all these examples to `tests/text/__init__.py` or create `tests/text/sample_inputs.py` and unify with all other samples inside text testing?

Reply:
It might be nice to store these examples outside of individual test files. I have no strong opinion on whether the example should be created into the `__init__.py` file or a new `sample_inputs.py` file.

Reply:
For `classification` we just have a `inputs.py` file in the corresponding folder.
I think it would be great to unify with the other metrics but lets do that in a follow up PR :]

Reply:
I created a separate issue for this idea. Here. I'll mark this conversation as resolved for now:]